src/auth/authenticationManager.ts

```diff
@@ -11,10 +11,10 @@ class AuthenticationManager {
       typeof error.data === 'object' &&
       error.data !== null &&
       'status' in error.data &&
-      typeof error.data.status === 'string'
+      typeof (error.data as any).status === 'string'
```

Comment:
would be great to add a comment here that this type casting happened because of TS version incompatibility with react-router. Otherwise we would not need to use this `any`.
This comment is needed in a real project to avoid unnecessary questions

Reply:
done